crates/ruff_linter/resources/test/fixtures/refurb/FURB142.py

```diff
@@ -0,0 +1,56 @@
+# Errors
+
+s = set()
+
+for x in [1, 2, 3]:
+    s.add(x)
+
+for x in {1, 2, 3}:
+    s.add(x)
+
+for x in (1, 2, 3):
+    s.add(x)
+
+for x in (1, 2, 3):
+    s.discard(x)
+
+for x in (1, 2, 3):
+    s.add(x + 1)
+
+for x, y in ((1, 2), (3, 4)):
+    s.add((x, y))
```

Comment:
Would you mind adding a test case where the value passed to add also depends on other variables other than the loop variable, like https://github.com/apache/airflow/blob/e41b3c58e2d2f5968bd97b8d3b05a16e926be980/airflow/utils/dag_edges.py#L74

Reply:
Added.
See [commit](https://github.com/astral-sh/ruff/pull/10583/commits/02fa99ebbec92bca7d20be09d95da270baf0e08b)

Reply:
Thanks